core/templates/dev/head/css/oppia.css

```diff
@@ -1055,7 +1055,7 @@ textarea {
   background: linear-gradient(to right, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%);
   /*The display is set in Library.js*/
   display: none;
-  height: 240px;
+  height: 259px;
```

Comment:
Re 258/259, I think I would prefer consistency. Is it possible to keep 258 and adjust the gradient/shadow?

It is very strange that the separate settings did not work. Can you show me what you did?